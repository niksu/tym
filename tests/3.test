   a2(b2).   a(b). 

   a1(b1). 
  a3(b3). 
