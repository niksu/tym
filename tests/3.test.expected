input contents |   a2(b2).   a(b). 

   a1(b1). 
  a3(b3). 
  a3(b4). 
|
(no query given)
stringed file contents (size=38, remaining=262)
|a2(b2).
a(b).
a1(b1).
a3(b3).
a3(b4).|
